server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -41,7 +41,8 @@ public class ClusterMetadataManifest implements Writeable, ToXContentFragment {
     public static final int CODEC_V0 = 0; // Older codec version, where we haven't introduced codec versions for manifest.
     public static final int CODEC_V1 = 1; // In Codec V1 we have introduced global-metadata and codec version in Manifest file.
     public static final int CODEC_V2 = 2; // In Codec V2, there are seperate metadata files rather than a single global metadata file.
```

Comment:
Add note that Code_V2 - never went live. 2.15 is going with V3 directly.

Reply:
Merged the new changes in Codec_V2 only, and removed V3